### ISSUE TITLE
HI

### ISSUE THREAD

**Issue by yulongPan** [jaiwei98/MobileNetV4-pytorch#7]
Hello, I would like to ask why I am using V4 classification with Arcface loss function and why the accuracy has always been 0. I only added a fully connected layer to the last layer and left it unmodified

**Comment by jaiwei98** [comment]
Hello, I assume you are doing binary classification.
Is that possible that you accidently reversed the output in your model? So that the accuracy always been 0.

**Comment by yulongPan** [comment]
> 您好，我假设您正在进行二元分类。您是否可能不小心反转了模型中的输出？因此，精度始终为 0。

Thank you for your reply. I used arcface loss with your V4 model, and I don't think I reversed the output. May I have your contact information

> Hello, I assume you are doing binary classification. Is that possible that you accidently reversed the output in your model? So that the accuracy always been 0.

Thank you for your reply. I used arcface loss with your V4 model, and I don't think I reversed the output. May I have your contact information

**Comment by yulongPan** [comment]
I had solve my problem，Thank you

**Comment by NewboySoCool** [comment]
> I had solve my problem，Thank you

How to solve?

**Comment by yulongPan** [comment]
你好，时间有点太久了，我有些忘记了，我应该是通过调整学习率进行优化的，还有一个就算好像是调整了某一层，奥对的，我想起来了，你可以检查一下你的Arcface的损失编写，它里面某一层应该出现了问题，当然，如果你愿意的话，可以分享给我

**Comment by yulongPan** [comment]
> > I had solve my problem，Thank you
> 
> How to solve?
你好，时间有点太久了，我有些忘记了，我应该是通过调整学习率进行优化的，还有一个就算好像是调整了某一层，奥对的，我想起来了，你可以检查一下你的Arcface的损失编写，它里面某一层应该出现了问题，当然，如果你愿意的话，可以分享给我

**Comment by NewboySoCool** [comment]
感谢回复，你有mobilenetv4的预训练权重？  除了timm上面那个，我用timm上面那个训练结果都有问题。 
















在 2025-03-19 16:26:21，"yulongPan" ***@***.***> 写道：

I had solve my problem，Thank you

How to solve?
你好，时间有点太久了，我有些忘记了，我应该是通过调整学习率进行优化的，还有一个就算好像是调整了某一层，奥对的，我想起来了，你可以检查一下你的Arcface的损失编写，它里面某一层应该出现了问题，当然，如果你愿意的话，可以分享给我

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***>

yulongPan left a comment (jaiwei98/MobileNetV4-pytorch#7)

I had solve my problem，Thank you

How to solve?
你好，时间有点太久了，我有些忘记了，我应该是通过调整学习率进行优化的，还有一个就算好像是调整了某一层，奥对的，我想起来了，你可以检查一下你的Arcface的损失编写，它里面某一层应该出现了问题，当然，如果你愿意的话，可以分享给我

**Comment by yulongPan** [comment]
> 感谢回复，你有mobilenetv4的预训练权重？  除了timm上面那个，我用timm上面那个训练结果都有问题。
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 在 2025-03-19 16:26:21，"yulongPan" ***@***.***> 写道：
> 
> I had solve my problem，Thank you
> 
> How to solve?
> 你好，时间有点太久了，我有些忘记了，我应该是通过调整学习率进行优化的，还有一个就算好像是调整了某一层，奥对的，我想起来了，你可以检查一下你的Arcface的损失编写，它里面某一层应该出现了问题，当然，如果你愿意的话，可以分享给我
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you commented.Message ID: ***@***.***>
> 
> yulongPan left a comment ([jaiwei98/MobileNetV4-pytorch#7](https://github.com/jaiwei98/MobileNetV4-pytorch/issues/7))
> 
> I had solve my problem，Thank you
> 
> How to solve?
> 你好，时间有点太久了，我有些忘记了，我应该是通过调整学习率进行优化的，还有一个就算好像是调整了某一层，奥对的，我想起来了，你可以检查一下你的Arcface的损失编写，它里面某一层应该出现了问题，当然，如果你愿意的话，可以分享给我
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you commented.Message ID: ***@***.***>

抱歉，我已经找不到了他的权重，因为我使用它在我的任务中效果没有达到预期，所以最后并未使用它

**Comment by NewboySoCool** [comment]
好的，感谢回复。我再试一试。感谢！
















在 2025-03-19 16:38:31，"yulongPan" ***@***.***> 写道：

感谢回复，你有mobilenetv4的预训练权重？ 除了timm上面那个，我用timm上面那个训练结果都有问题。

在 2025-03-19 16:26:21，"yulongPan" @.***> 写道：

I had solve my problem，Thank you

How to solve?
你好，时间有点太久了，我有些忘记了，我应该是通过调整学习率进行优化的，还有一个就算好像是调整了某一层，奥对的，我想起来了，你可以检查一下你的Arcface的损失编写，它里面某一层应该出现了问题，当然，如果你愿意的话，可以分享给我

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: @.***>

yulongPan left a comment (jaiwei98/MobileNetV4-pytorch#7)

I had solve my problem，Thank you

How to solve?
你好，时间有点太久了，我有些忘记了，我应该是通过调整学习率进行优化的，还有一个就算好像是调整了某一层，奥对的，我想起来了，你可以检查一下你的Arcface的损失编写，它里面某一层应该出现了问题，当然，如果你愿意的话，可以分享给我

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: @.***>

抱歉，我已经找不到了他的权重，因为我使用它在我的任务中效果没有达到预期，所以最后并未使用它

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***>

yulongPan left a comment (jaiwei98/MobileNetV4-pytorch#7)

感谢回复，你有mobilenetv4的预训练权重？ 除了timm上面那个，我用timm上面那个训练结果都有问题。

在 2025-03-19 16:26:21，"yulongPan" @.***> 写道：

I had solve my problem，Thank you

How to solve?
你好，时间有点太久了，我有些忘记了，我应该是通过调整学习率进行优化的，还有一个就算好像是调整了某一层，奥对的，我想起来了，你可以检查一下你的Arcface的损失编写，它里面某一层应该出现了问题，当然，如果你愿意的话，可以分享给我

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: @.***>

yulongPan left a comment (jaiwei98/MobileNetV4-pytorch#7)

I had solve my problem，Thank you

How to solve?
你好，时间有点太久了，我有些忘记了，我应该是通过调整学习率进行优化的，还有一个就算好像是调整了某一层，奥对的，我想起来了，你可以检查一下你的Arcface的损失编写，它里面某一层应该出现了问题，当然，如果你愿意的话，可以分享给我

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: @.***>

抱歉，我已经找不到了他的权重，因为我使用它在我的任务中效果没有达到预期，所以最后并未使用它

**Comment by jaiwei98** [comment]
Weight released in latest commit! Thank you for interested in this project :D

**Comment by NewboySoCool** [comment]
Thanks! Good jobs!













At 2025-03-19 21:38:30, "jaiwei" ***@***.***> wrote:

Weight released in latest commit! Thank you for interested in this project :D

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***>

jaiwei98 left a comment (jaiwei98/MobileNetV4-pytorch#7)

Weight released in latest commit! Thank you for interested in this project :D
